src/status_im2/contexts/wallet/events.cljs

```diff
@@ -0,0 +1,22 @@
+(ns status-im2.contexts.wallet.events
+  (:require [taoensso.timbre :as log]
+            [re-frame.core :as re-frame]))
```

Comment:
[utils.re-frame :as rf]

Reply:
I did it as part of https://github.com/status-im/status-mobile/pull/17451